index.html

```diff
@@ -105,7 +128,7 @@ <h3 class="text_titleH3">Project 3</h3>
             <div class="container-work" id="project4">
```

Comment:
[OPTIONAL]

You're missing a background image here: 

- Your project: 
![image](https://user-images.githubusercontent.com/78335447/171969235-3a6a63b3-6186-489a-979e-3666980d2133.png)

- Figma's: 
![image](https://user-images.githubusercontent.com/78335447/171969260-05fbd8e6-d08c-47bc-90ab-141417e09665.png)